content/pages/mua-summary.md

```diff
@@ -0,0 +1,68 @@
+Title: MUA Summary
+Category: pages
+Status: hidden
+Slug: mua-summary
+
+Anyone with a user account to a WiCS machine must sign our [Machine Usage Agreement]({filename}/pages/mua.md),
+summarized below. Those with guest accounts do not need to sign the MUA, but they are still held
+to the rules and regulations outlined in it.
+
+# Acceptable and Unacceptable Use #
+
+The WiCS machines are to be used for learning, class work, personal projects, and 
+use in accordance with [WiCS aims]({filename}/pages/about.md), that does not violate
+any of WiCS’ usage policies.
+
+If you have a project of interest that does not fall into these categories, you may
+request special priority from wics-sys.
+
+# User Responsibilities #
+
+All usage on WiCS machines is governed by our [CoC]({filename}/pages/code-of-conduct.md), 
+our [MUA]({filename}/pages/mua.md), local laws, and University policy. Users are also
+responsible for being aware of and adhering to policy of machines connected to from 
+WiCS machines.
```

Comment:
Maybe add that "This means, among other things, that machines may not be used for commercial purposes, including the hosting of corporate websites and mining cryptocurrency."